README.md

```diff
@@ -0,0 +1,49 @@
+# URLess
+
+[![GitHub license](https://img.shields.io/github/license/Naereen/StrapDown.js.svg)](https://github.com/Naereen/StrapDown.js/blob/master/LICENSE)
+
+Shorten (or slightly lengthen until we get a shorter domain) your URLs in one easy click!
+
+## Installation & Usage
+
+Visit [URLess](https://urlessen.herokuapp.com/)!
+
+Or clone/download the repo and:
+
+* Open a terminal and navigate into the folder
+* Run `pipenv install -r requirements.txt` to install dependencies
+* Navigate into /shortener/ and run `./manage.py runserver` to start locally
+
```

Comment:
Nice clear instructions! On a minor note, did you know that adding a script such as `dev = "./shortener/manage.py run server"` to your pipfile would allow your user to simply `pipenv run dev` without `cd`ing anywhere ...